compose/foundation/foundation-layout/src/androidAndroidTest/kotlin/androidx/compose/foundation/layout/RowColumnTest.kt

```diff
@@ -1692,6 +1692,34 @@ class RowColumnTest : LayoutTest() {
         assertTrue(latch.await(1, TimeUnit.SECONDS))
```

Comment:
@eymar there were simple merge conflicts: 
  * androidTests
  * build.gradle where version number of a library was changed
  * import of kotlin.jvm.JvmInline in InlineDensityKt in commonMain (this should be upstreamed)

<img width="936" alt="image" src="https://github.com/JetBrains/compose-multiplatform-core/assets/23081446/49668c5a-d841-4db5-bb87-d4360f135dd7">